script.module.libmediathek4/addon.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.libmediathek4" name="libmediathek4" version="1.0.0" provider-name="sarbes">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0" />
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+    <source>https://github.com/sarbes/plugin.video.libmediathek4</source>
+    <summary lang="en">Supplementary lib for Mediathek add-ons.</summary>
+    <description lang="en">This lib is a mini framework. It is used in my (sarbes) "Mediathek" scrapers.</description>
+  </extension>
```

Comment:
missing assets? Although this is a script.module it will still be shown in the UI once you install an addon that depends on it. Can you add an icon? You can use one default (look for other script.modules in the repo)